src/encoded/static/components/image.js

```diff
@@ -5,177 +5,149 @@ import * as globals from './globals';
-                        </div>
-                        <Lightbox lightboxVisible={this.state.lightboxVisible} lightboxImg={attachmentHref} clearLightbox={this.clearLightbox} />
-                    </div>
-                );
-            } else if (attachment.type === 'application/pdf') {
-                // Attachment is a PDF. Show the PDF icon, and clicks in it show the PDF in a new
-                // window using the browser's PDF viewer.
-                return (
+            // Display the attachment image in a light box.
+            return (
+                <div>
                     <div className="attachment">
-                        <a data-bypass="true" href={attachmentHref} className="attachment__button" target="_blank" rel="noopener noreferrer" title="Open attachment in a new window">
+                        <a className="attachment__button" data-bypass="true" href={attachmentHref} onClick={lightboxClickImage} title="View attachment in this window">
                             <div className="attachment__hover" />
-                            <div className="file-pdf">Attachment PDF Icon</div>
+                            <img className={imgClass} src={src} height={height} width={width} alt={alt} />
                         </a>
                     </div>
-                );
-            }
-
-            // Non image, non PDF attachment (likely a text file). Show the generic document icon
-            // and open it in a new window.
+                    <Lightbox lightboxVisible={lightboxVisible} lightboxImg={attachmentHref} clearLightbox={clearLightbox} />
+                </div>
+            );
+        } else if (attachment.type === 'application/pdf') {
+            // Attachment is a PDF. Show the PDF icon, and clicks in it show the PDF in a new
+            // window using the browser's PDF viewer.
             return (
                 <div className="attachment">
                     <a data-bypass="true" href={attachmentHref} className="attachment__button" target="_blank" rel="noopener noreferrer" title="Open attachment in a new window">
                         <div className="attachment__hover" />
-                        <div className="file-generic">Attachment Icon</div>
+                        <div className="file-pdf">Attachment PDF Icon</div>
                     </a>
                 </div>
             );
         }
 
-        // No attachment given; display a broken file icon.
+        // Non image, non PDF attachment (likely a text file). Show the generic document icon
+        // and open it in a new window.
         return (
             <div className="attachment">
-                <div className="attachment__button">
-                    <div className="file-missing">Attachment file broken icon</div>
-                </div>
+                <a data-bypass="true" href={attachmentHref} className="attachment__button" target="_blank" rel="noopener noreferrer" title="Open attachment in a new window">
```

Comment:
is data-bypass still needed?

Reply:
I really have no idea. We use it all over the place but I don’t remember what it’s for. It might be worth a ticket to investigate, and remove all uses of it if we don't need it.